pkg/upgrade/v1alpha2/kubernetes_test.go

```diff
@@ -679,68 +679,83 @@ func TestUpgradeTaskPatch(t *testing.T) {
 
 func TestGetUpgradeDetailedStatuses(t *testing.T) {
```

Comment:
Change func name to it's test functionality.

Reply:
Done